### ISSUE TITLE
why no --force option for manjaro/arch?

### ISSUE THREAD

**Issue by AntiNSA** [teejee2008/aptik#12]
Hi! This is a huge issue.Where is the --force option?

like packan -S --force packagename ?

Thanks if I am missing something. All my restores dont work due to preexisting files from broken installs. I want to --force and I cant figure out how.

**Comment by yochananmarqos** [comment]
Aptik isn't designed for broken installs. It's designed to restore packages after a clean install. Using `--force` can be a dangerous option and is never recommended.

**Comment by AntiNSA** [comment]
It would be nice if it was a user option, no matter the danger. It's a user
option in Linux.

**Comment by AntiNSA** [comment]
Also, the problem is if you have a lot of packages to restore, one or two
pre-existing icons, can stop all packages from being installed. This not
the case in Aptik with Ubuntu. I have used April for now more than a year
to provide support to about 30 students laptops. Now I am changing from
Ubuntu to Manjaro, and having this restore problem that is nearly making
Aptik useless, as packages will not be restored at all if there is any
images in the pre-existing template folders. As a user who has heavily used
Aptik,I am kindly asking you to create an allow --force option as a user
option. Without that I will have to choose a different solution.

**Comment by teejee2008** [comment]
Aptik should not be used to move from one distribution type (Ubuntu, based on Debian) to another distribution type (Manjaro, based on Arch Linux). This is stated clearly on the [project page](https://github.com/teejee2008/aptik/blob/master/README.md).

Different distributions will have different names for the same packages. Trying to reinstall packages and repos meant for one distribution on another unrelated distribution is not logical and not supported by Aptik. 

Please go through the Readme and let me know if you have any questions.

**Comment by teejee2008** [comment]
Please note that you can still restore themes, fonts, users, etc. Its only repos and packages that are not supported.

**Comment by AntiNSA** [comment]
I dont think you should close this. I did not mean I was trying to restore from one distro to another.

I said to you that I had used aptik previously with Ubuntu extensively to support multiple teams of students.

Now that I have changed platforms, from Ubuntu to a new platform, I would like to continue using it with a new platform...I understand what you are saying.

What I am srying is that in Ubuntu, there was no issue with restoring packages ,,,, however in manjaro I am unable to use it the same way.....

I am talking about manjaro/manjaro   ... 

In ubuntu/ubuntu I have  no need for the --force option....

So basically I hope you can add the option to --force installations.

Repos and packages are the most urgent neccessity.

Unfortunately I can not continue to use Ubuntu only for aptik. 


I hope wiht a slight alteratioin of aptik I can use it with Manjaro. I just need the option of --force to be there.

I understand the risk.

Let the user chose the risk.

**Comment by teejee2008** [comment]
Sorry, I misunderstood what you were trying to do. I'll see if I can add the option somewhere.
